### ISSUE TITLE
Crash when favoriting

### ISSUE THREAD

**Issue by eykamp** [jeffvli/feishin#672]
### Expected Behavior

I have the standard three panes that are visible when the program is first installed.  I click on Albums on the left pane, then on the heart icon on an album in the center pane.  The screen turned white and locked.  This happened three times in a row, so it seems pretty reproducible.

![image](https://github.com/user-attachments/assets/fb46ebd6-940a-4142-847a-b9c645c41b5a)

### Current Behavior

No crash.

### Steps to Reproduce

See above

### Application version

0.7.1

### Operating System and version

Win10

### Server and Version

Navidrome 0.49.3 (8b93962f)

**Comment by iiPythonx** [comment]
If you don't have a reason to stay on 0.49.3, maybe try updating to the latest version of Navidrome and see if that fixes the issue? I'm not able to reproduce this on 0.52.0.

**Comment by eykamp** [comment]
I'll try it with Navidrome 0.52, but I think it's a front end issue because favoriting through other mechanisms works just fine.  I assume that no matter how you favorite an album, it hits the same Navidrome endpoint with the same data.

But I'll try again and let you know.

Thanks!

**Comment by matthilde** [comment]
Hello! I am having the same issue with Feishin when I try to favourite an album specifically in the "Albums" section. For some reason faovuriting on the main page does not do anything.

I get the following log when running the app from a terminal
```
13:14:23.625 › [AUDIO PLAYER] Setting properties: {"speed":1}
[32804:0719/131452.928298:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[32804:0719/131453.930306:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
[32804:0719/131454.932956:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
```

Running Feishin 0.7.1 with Navidrome 0.52.

**Comment by jeffvli** [comment]
@matthilde Those logs are only relevant to the executable process rather than the app itself. From what I can see there's an issue with the cached data not invalidating itself when an item is favorited on the home page. The network request to favorite the item will still go through though, it just doesn't reflect when navigating from and back to the homepage.

@eykamp If you can reproduce this with console logs (you can open the network devtools from the app menu or with `CTRL` + `SHIFT` + `I` then we can potentially fix this.

**Comment by eykamp** [comment]
I can still reproduce.  I get the following 2 errors in the console when it happens:

```
renderer.js:2 TypeError: Cannot read properties of undefined (reading 'id')
    at renderer.js:1958:511
    at Array.map (<anonymous>)
    at renderer.js:1200:1319
    at _i (renderer.js:2:2473840)
    at bi (renderer.js:2:2474298)
    at Object.useState (renderer.js:2:2480674)
    at e.useState (renderer.js:2:17280438)
    at renderer.js:1200:612
    at yi (renderer.js:2:2473022)
    at Hl (renderer.js:2:2483081)
```
```
ul @ renderer.js:2
renderer.js:2 Uncaught TypeError: Cannot read properties of undefined (reading 'id')
    at renderer.js:1958:511
    at Array.map (<anonymous>)
    at renderer.js:1200:1319
    at _i (renderer.js:2:2473840)
    at bi (renderer.js:2:2474298)
    at Object.useState (renderer.js:2:2480674)
    at e.useState (renderer.js:2:17280438)
    at renderer.js:1200:612
    at yi (renderer.js:2:2473022)
    at Hl (renderer.js:2:2483081)
```

**Comment by kgarner7** [comment]
This should be resolved with https://github.com/jeffvli/feishin/commit/c5e8472746617abaa72fb26c92d8f7644f61e07d (and 0.7.3). Is this issue still being seen?

**Comment by kgarner7** [comment]
Since I haven't heard any updates, I'll mark this as resolved. If it is still seen, please add more details and verify it on https://feishin.vercel.app as well for latest

**Comment by eykamp** [comment]
Confirming that the issue appears to be fixed in 0.7.3.  Thanks!
